test/e2e/e2e_test.go

```diff
@@ -0,0 +1,106 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/engine"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/runtime"
+)
+
+// These tests confirm that all container and operator policies properly pass
+// with a known-good image, and properly fail with a known-bad image.
+// Any check that is found in the error section of the Result will cause this
+// to fail.
+var _ = Describe("policy validation", func() {
+	Describe("When enforcing operator policy", func() {
+		var (
+			// TODO: replace this with a project-accessible namespace once identified.
+			goodImage = "quay.io/komish/preflight-test-bundle-passes:latest"
+			badImage  = "quay.io/komish/preflight-test-bundle-fails:latest"
+		)
+
+		Context("with a known-good image", func() {
+
+			cfg := runtime.Config{
+				Image:         goodImage,
+				EnabledChecks: engine.OperatorPolicy(),
+			}
+
+			engine, err := engine.NewForConfig(cfg)
+			Expect(err).To(BeNil())
+
+			engine.ExecuteChecks()
+			results := engine.Results()
+
+			It("should pass all checks", func() {
+				Expect(len(results.Passed)).To(Equal(len(cfg.EnabledChecks)))
```

Comment:
for the sake of assurance, could you please assert that the number of failed/errored tests are 0?

Reply:
This should be resolved!